autopilot/contractor/contractor.go

```diff
@@ -658,6 +622,9 @@ func renewFundingEstimate(minRenterFunds, initRenterFunds, remainingRenterFunds
 		// otherwise we use the remaining funds from before because a renewal
 		// shouldn't add more funds, that's what a refresh is for
 		renterFunds = remainingRenterFunds
+		if minFunds := initRenterFunds.Mul64(8).Div64(10); renterFunds.Cmp(minFunds) < 0 {
```

Comment:
Doesn't it make more sense to use `initRenterFunds` if `remainingRenterFunds` is less than 50% of the initial funds? I feel that's slightly less arbitrary and it matches with halving the funds when we haven't used the contract.